### DISCUSSION TITLE
Embed HTML into the pdf

### DISCUSSION THREAD

**Discussion by FedericaCabrera1** [Hopding/pdf-lib#1144] [Q&A]
Hey there! I wanted to ask for your help in order to solve a problem im encountering. Im trying to embed an HTML string into the report. 
Context: I'm using a wysiwyg library that returns me a string with the HTML tags, and would like to render this inside the pdf. 

Do you have any idea / recommendation in order to solve this? 
Thanks!

**Comment by yaakovyitzchak** [comment]
BH

In most cases u can easily convert HTML to an SVG, and an SVG to a PNG using basic native javascript, then u can embed that image in the PDF

see attached file for example implementation
[BH_HTMLtoImage.zip](https://github.com/Hopding/pdf-lib/files/8803277/BH_HTMLtoImage.zip)
